source/includes/_users.md

```diff
@@ -574,3 +574,81 @@ DONE | Delivery is done
 UNKNOWN | Delivery status is unknown
 QUEUED | Delivery is in queue
 
+
+## List All Payment Methods for a User
+
+> Definition
+
+```
+GET https://api.kvass.ai/users/<user-id>/payment_methods
+```
+
+> Example request:
+``` http
+GET /users/<user-id>/payment_methods HTTP/1.1
+Content-Type: application/json
+Authorization: Bearer <jwt>
+X-Kvass-Api-Key: <kvass-api-key>
+Host: api.kvass.ai
+```
+
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+[
+    {
+       "_id":{"$oid":<payment-method-id>},
+       "_cls":"PaymentMethod.StripePaymentMethod",
+       "created":{"$date":1538058324877},
+       "modified":{"$date":1538058327405},
+       "deleted":false,
+       "active":true,
+       "user":{"$oid":"57ee9c72d76d431f85111432"},
+       "company":{"$oid":"591ee15db70e2a10acb65362"},
+       "name":"Stripe",
+       "method":"stripe",
+       "token":"tok_1DF0QiEeeXxFpLJtthyCQWVz",
+       "customer_id":"cus_DgNBPtNphW2W7r",
+       "card":{
+          "id":"card_1DF0QiEeeXxFpLJtg4AE1T5z",
+          "object":"card",
+          "address_city":null,
+          "address_country":null,
+          "address_line1":null,
+          "address_line1_check":null,
+          "address_line2":null,
+          "address_state":null,
+          "address_zip":null,
+          "address_zip_check":null,
+          "brand":"Visa",
+          "country":"US",
+          "cvc_check":"unchecked",
+          "dynamic_last4":null,
+          "exp_month":12,
+          "exp_year":2019,
+          "fingerprint":"npPw68vQg1usKUWb",
+          "funding":"credit",
+          "last4":"4242",
+          "metadata":{},
+          "name":null,
+          "tokenization_method":null
+       }
+    }
+]
+```
+
+Retrieves a list of all [Payments Method](#payment_methods) associated with a given user id.
+This route will work a bit differently if the authenticated user is an admin or a regular costumer.
+In case of a regular user/costumer, the user could execute this request only with his own `user id`. If he tries with another `user id` then the API will return an `403 error`.
+However, an `admin` user can search by any `user id`.
```

Comment:
I suggest leaving out lines 641 - 643.  Hackers will try, then get a 403... And admins will assume that it works because they are admins

Reply:
Okey